.travis.yml

```diff
@@ -11,6 +11,11 @@ env:
     - IMAGE_TAG="noobaa-${TRAVIS_COMMIT}"
     - TESTER_TAG="noobaa-tester-${TRAVIS_COMMIT}"
 
+    - NO_CACHE="NO_CACHE=true"
+    - SUPPRESS_LOGS="SUPPRESS_LOGS=true"
+    - if [[ $TRAVIS_PULL_REQUEST_BRANCH =~ "dbg" ]]; then
```

Comment:
@liranmauda What does this check mean? How can it be used? Maybe worth a code comment?

Reply:
Just saw the answer in the PR title, better have it in the code too I think

Reply:
@guymguym 
`[[ $TRAVIS_PULL_REQUEST_BRANCH =~ "dbg" ]]` is:
if the variable contain the string, probebly better to use:
`[[ "$TRAVIS_PULL_REQUEST_BRANCH" =~ "dbg" ]]`
I will update that

Reply:
But is this variable containing the PR headline? Does headline edits work if I want to add debug?

Reply:
apparently no, this method doesn't work as the $TRAVIS_PULL_REQUEST_BRANCH doesn't contain the PR name.